runtime/exec/environment.go

```diff
@@ -18,27 +18,27 @@ import (
 	"github.com/segmentio/ksuid"
 )
 
-type Source struct {
+type Environment struct {
 	engine storage.Engine
 	lake   *lake.Root
 }
 
-func NewSource(engine storage.Engine, lake *lake.Root) *Source {
-	return &Source{
+func NewEnvironment(engine storage.Engine, lake *lake.Root) *Environment {
+	return &Environment{
 		engine: engine,
 		lake:   lake,
 	}
 }
 
-func (s *Source) IsLake() bool {
+func (s *Environment) IsLake() bool {
```

Comment:
Nit: Update receiver name on all these methods.
```suggestion
func (e *Environment) IsLake() bool {
```